governance/GOVERNANCE.md

```diff
@@ -76,6 +76,51 @@ New ORAS owners participate in an onboarding period during which they fulfill al
 
 The onboarding period is intended to ensure that the to-be-appointed owner is able/willing to take on the time requirements, familiar with ORAS core logic and concepts, understands the overall system architecture and interactions that comprise it, and is able to work well with both the existing owners and the community.
 
+## Working Groups
+
+Working Groups are primarily used to facilitate topics of discussion that are in scope for ORAS but that cross multiple sub-projects and teams. It provides a formal avenue for disparate teams or groups to collaborate around a common problem, craft a balanced position, and disband.
+
+The Working Group does not own any code. To create a Working Group in the ORAS community, organizers should have a clear goal measured through a specific deliverable or deliverables. Working Group is temporary in nature and will be eventually disbanded after reaching its stated goal(s). 
```

Comment:
```suggestion
To create a Working Group in the ORAS community, organizers should have a clear goal measured through a specific deliverable or deliverables. Working Group is temporary in nature and will be eventually disbanded after reaching its stated goal(s).
```

Reply:
The major differentation Sajay makes above is:

> temporary in nature and will be eventually disbanded after reaching its stated goal(s).

Is that effectively the only difference between a sub-project and working group? 

For instance:

> To create a Working Group in the ORAS community, organizers should have a clear goal measured through a specific deliverable or deliverables.

Wouldn't that also apply to sub-projects:

> To create a Subproject in the ORAS community, maintainers should have a clear goal measured through a specific deliverable or deliverables.

I might suggest a working group may not have clear goals. That's kinda the point. It's a place of incubation, to explore some ideas, like a startup that starts with one idea with a sore set of like-minded people. They may pivot a few times, and that's ok, and actually, the freedom to land a great user-focused solution. 

Even sub-projects pivot as they learn and explore new scenarios. I think we should be careful to not overly limit the creativity. We might want to put some scope barriers up, only to set expectations to come back to the "org owners" for approval to expand charter.